### DISCUSSION TITLE
Possible hack of a link on BEST-toolbox.org

### DISCUSSION THREAD

**Discussion by Fransina-King** [umair-hassan/best-toolbox#43] [General]
Hi Umair,

I just wanted to inform you that it seems like the website linked in step 2.1.3, instructions for bossdevice firmware 2017b, has been hacked. I've attached a screenshot below.
![image](https://user-images.githubusercontent.com/113104819/189102913-747d4b30-12d5-4ca7-90c5-05a3983c0314.png)

**Comment by umair-hassan** [comment]
Hi Fransina,

Thankyou. I will redirect the website, its pointing to old page.

Thanks again

**Comment by Fransina-King** [comment]
Hi Umair,

You're welcome.
